skywalking/__init__.py

```diff
@@ -44,6 +44,7 @@ class Component(Enum):
     Psycopg = 7010
     Celery = 7011
     Falcon = 7012
+    MysqlClient = 7013
```

Comment:
7013 is not registered in the main repo, it is better to do that before merging.